src/button/validate.js

```diff
@@ -22,7 +22,7 @@ export function validateButtonLocale(locale : string) {
     }
 }
 
-export function validateButtonStyle(style : Object = {}) {
+export function validateButtonStyle(style : Object = {}, props : Object ) {
```

Comment:
Can we make this more specific?

```javascript
export function validateButtonStyle(style : Object = {}, { locale } : { locale : string }) {
```

Reply:
can be done. However, we may need other props ( like locale this time) for the purpose of button validation.